### ISSUE TITLE
LuckPerms Not Supported

### ISSUE THREAD

**Issue by Hazard-The-AD** [Creeperface01/PlaceholderAPI-nukkit#7]
Hey Creeperface.

I've tried using your plugin to work with LuckPerms and we haven't had any luck. We are using the correct placeholders for example: %luckperms_prefix%, although it doesn't switch to the player's prefix, it just shows the placeholder name. We are using a plugin called Simple Scoreboards which allows for your placeholders although there wasn't a command called /papi, it just said unknown command when I tried to run it.
![ApplicationFrameHost_80RkpXBVqH](https://user-images.githubusercontent.com/20299632/80306527-c8758300-87bb-11ea-84da-12f23d2b69a7.png)
![ApplicationFrameHost_0EtRuReYCT](https://user-images.githubusercontent.com/20299632/80306528-ca3f4680-87bb-11ea-93a2-0318e268ab14.png)

This is along with the latest version of NukkitX (not 2.0)
Hopefully, we can get this fixed as we really need this plugin working as it is going to play a key part within our network.

**Comment by Hazard-The-AD** [comment]
![ApplicationFrameHost_X2ht5ixZVo](https://user-images.githubusercontent.com/20299632/80306609-420d7100-87bc-11ea-9a9f-8b6437a5f8b5.png)
![ApplicationFrameHost_ZljgOMCVf3](https://user-images.githubusercontent.com/20299632/80306611-43d73480-87bc-11ea-8d9d-54a6e81c3cb1.png)

In addition, this is the only command that shows what we can do.

**Comment by Creeperface01** [comment]
Yes I know about this bug and its fixed in 1.4 and I'm not sure whether it's worth it to break existing plugins before the 2.0 however if you need it only for your own plugins you could use the 1.4 branch

**Comment by Hazard-The-AD** [comment]
Thanks so much for your swift reply. We'll try and use 1.4 and see if that changes anything. Thanks for the help.

**Comment by Creeperface01** [comment]
Note that 1.4 breaks the compatibility with all current plugins supporting
PAPI

**Comment by Creeperface01** [comment]
wait, and do you have MorePlaceholders plugin installed? LuckPerms doesn't have builtin placeholder support
